arbeitszeit_flask/views/signup_company_view.py

```diff
@@ -34,7 +35,7 @@ def GET(self) -> Response:
         return render_template("auth/signup_company.html", form=register_form)
 
     @commit_changes
-    def POST(self) -> Response:
+    def POST(self) -> Response | Tuple[Response, int]:
```

Comment:
Yes great I saw that around in the http_error_view.py file while I was now looking around and was thinking about that, looks a lot better with the typing now (it can go back to returning the same Response type as before so not some weird exception).